src/amo/components/MoreAddonsByAuthorsCard/index.js

```diff
@@ -0,0 +1,203 @@
+/* @flow */
+import makeClassName from 'classnames';
+import deepEqual from 'deep-eql';
+import * as React from 'react';
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+
+import AddonsCard from 'amo/components/AddonsCard';
+import {
+  fetchAddonsByAuthors,
+  getAddonsForUsernames,
+  getLoadingForAuthorNames,
+} from 'amo/reducers/addonsByAuthors';
+import {
+  ADDON_TYPE_DICT,
+  ADDON_TYPE_EXTENSION,
+  ADDON_TYPE_LANG,
+  ADDON_TYPE_THEME,
+} from 'core/constants';
+import { withErrorHandler } from 'core/errorHandler';
+import translate from 'core/i18n/translate';
+import type { AddonsByAuthorsState } from 'amo/reducers/addonsByAuthors';
+import type { ErrorHandlerType } from 'core/errorHandler';
+import type { AddonType } from 'core/types/addons';
+import type { I18nType } from 'core/types/i18n';
+import type { DispatchFunc } from 'core/types/redux';
+
+import './styles.scss';
+
+
+type Props = {|
+  addons?: Array<AddonType> | null,
+  addonType?: string,
+  authorNames: Array<string>,
+  className?: string,
+  dispatch: DispatchFunc,
+  errorHandler: ErrorHandlerType,
+  forAddonSlug?: string,
+  i18n: I18nType,
+  loading?: boolean,
+  numberOfAddons: number,
+
+  // AddonCards prop this component also accepts
+  showSummary?: boolean,
+  type?: 'horizontal',
```

Comment:
This is actually copied from elsewhere and is just the values this component (and the component it passes things to) accepts. The idea is instead of constants we can just say the values it accepts in the type.

Reply:
It does nothing with any other value than `undefined` or `'horizontal'`, so leaving this as-is will make flow complain if we pass it anything else, which is intended.

Reply:
Ah, I assumed it could accept other types. What's the point of it accepting a prop if only one possible value is valid?

Reply:
The underlying component can have a horizontal type set or none set, and they'll be different. In the future we might end up having more types.

Arguably there should be a default set that isn't `undefined` but I think that's a code quality bug outside this patch.